Cargo.toml

```diff
@@ -62,3 +62,7 @@ harness = false
 [[bench]]
 name = "cctv"
 harness = false
+
+[patch.crates-io]
+# https://github.com/rust-random/core/pull/17
+rand_core = { git = "https://github.com/rust-random/core.git" }
```

Comment:
Can you pin the revision here? Don't want some new commits to break compilation.